src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/SubWriterHolderWriter.java

```diff
@@ -136,25 +135,22 @@ protected void addIndexComment(Element member, List<? extends DocTree> firstSent
     /**
      * Add the summary link for the member.
      *
-     * @param mw the writer for the member being documented
      * @param member the member to be documented
      * @param contentTree the content tree to which the link will be added
      */
-    public void addSummaryLinkComment(AbstractMemberWriter mw, Element member, Content contentTree) {
+    public void addSummaryLinkComment(Element member, Content contentTree) {
         List<? extends DocTree> tags = utils.getFirstSentenceTrees(member);
-        addSummaryLinkComment(mw, member, tags, contentTree);
+        addSummaryLinkComment(member, tags, contentTree);
```

Comment:
I find this (a bit) surprising, since it is (potentially) useful to have access to the enclosing writer, but that being said, if we ever really need access to the enclosing `AbstractMemberWriter` it is probably better passed in to the constructor.